### PR TITLE
488 472 remove group name alias and processes from device

### LINKED ISSUE #472: remove 'groupNameAlias' property from the device model
the property `groupNameAlias` on the device is currently not used. Remove it from the model and update the devices in the database.

afterwards, remove from devices in all environments!

There may be the skeleton of scripts to already do this.

### LINKED ISSUE #488: Deprecate the `processes` property in the device
blocked by: https://github.com/ospin-web-dev/REACTor/issues/748

The `processes` array refers to processes associated with the device. Because the processes also already point to their device this reference is not needed anymore. 

- [ ] remove it from the device model
- [ ] remove it from the devices in the database
- [x] fix all tests and other references within the code

### PR DESCRIPTION
**Pull request by vdiezel**
I migrated the database and was able to remove the props from the schema now.

closes #488 
closes #472

### CODE REVIEW

**Comment by DanielSeehausen** [comment]
(clicked auto update for you since this is the next thing that gets merged most likely)

File: lib/models/Device/Device.js [hunk 1]
```diff
@@ -32,8 +32,6 @@ class Device {
       streams: Joi.array().default([]),
       ownerId: Joi.string().pattern(UUIDV4_REGEX).default(null).allow(null),
       hardwareVersion: Joi.number().integer().default(0),
-      processes: Joi.array(), // TODO remove once the property was removed in the DB
-      groupNameAlias: Joi.array(),
```
Comment (DanielSeehausen) [inline lib/models/Device/Device.js:36]: 🔫

File: scripts/connectAllScripts.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  RemoveField,
+  PutField,
+]
+
+module.exports = async () => {
+  const db = await mongo.getDB()
+
+  return CONNECTABLE_SCRIPTS.map(connectable => (
+    connectable.connect(db)
+  ))
```
Comment (DanielSeehausen) [inline scripts/connectAllScripts.js:15]: nice

**Comment by ConorTighe1995** [comment]
I clicked re-request review by mistake so just ignore that @DanielSeehausen

**Pull request merged**